rholang/src/test/scala/coop/rchain/rholang/interpreter/NormalizeTest.scala

```diff
@@ -79,7 +82,32 @@ class CollectMatcherSpec extends FlatSpec with Matchers {
     )
     result.knownFree should be(inputs.knownFree)
   }
-
+  "List" should "sort elements" in {
```

Comment:
`s/sort elements/sort the insides of their elements`. Otherwise it suggests the list itself should be sorted, which this (nor the other tests) don't check. Please update all the new tests' names.

Reply:
Done